ang/crmMailing/BlockMailing.html

```diff
@@ -79,5 +79,19 @@
         placeholder="Subject"
         name="subject" />
     </div>
+    <div ng-if="crmMailingConst.isMultiLingual">
+      <div crm-ui-field="{name: 'subform.language', title: ts('Language')}">
+        <select
+          crm-ui-id="subform.language"
+          crm-ui-select="{dropdownAutoWidth : true, allowClear: false}"
+          name="language"
+          ng-model="mailing.language"
+          required
+          >
+          <option value=""></option>
```

Comment:
Yup I agree, either we need to set it to current UI language and remove empty option like the one I've shown in the screenshot  OR we should provide a placeholder like  ``` -- choose language -- ``` , and its ok to render blank option with such placeholder title because there is no way we will be able to submit our mailing data without choosing any language. It's ur call :)